modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-configuring-storage-class-bootsource-update_{context}"]
+= Configuring a storage class for custom boot source updates
+
+You must configure a storage class that allows automatic updating for custom boot sources.
+
+.Procedure
+
+* Choose the appropriate method to configure a storage class:
```

Comment:
Is this really a one-step procedure? 

It looks like two ways (edit HCO, patch storageclass) to do the same thing. If we provide two ways to do the same thing, normally we either tell the user how the procedures differ (example: situation requires y instead of z) or we just document the recommended way.

Reply:
I actually prefer just telling the user one way to do things, so I'm all for dropping one method if it's not the one most users would use. I think an audience on this level of sophistication could think of the `oc` commands themselves :-)

Do you feel strongly enough about this to pull in the SME? I will, if so.

Reply:
Please do!